### PR TITLE
Create risks.md

### PR DESCRIPTION
**Pull request by RByers**
Copied directly from https://github.com/RByers/identity-credential/blob/risks/risks.md. To be consolidated with credentials-considerations.md

### CODE REVIEW

File: risks.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+In addition, some people are unwilling or unable to rely on government-issued identification. Whether due to being in an at-risk demographic such as the unsheltered, out of a philosophical conviction, or just being too young, a dependence on RWI systems may place these users at risk of increasingly being excluded from larger fractions of online activity.
+
+### Widening the use of high-assurance identity beyond user expectations
+
+When a user has a legitimate need to use their high-assurance identity, there's a risk of that information getting used for scenarios broader than they intended. For example, using a government ID to enable consumption of content not intended for children resulting in a unique identifier being joined to the user's social media profile for the purposes of more effective ad targeting. Additionally, a security breach of a website with legitimate need for RWI could lead to greater consequences for identity theft or the trade of personal user information on the black market.
```
Comment (OR13) [inline risks.md]:
```suggestion
When a user has a legitimate need to use their high-assurance identity, there's a risk of that information getting used for scenarios broader than they intended. For example, using a government ID to enable consumption of content not intended for children resulting in a unique identifier being joined to the user's social media profile for the purposes of more effective ad targeting. Additionally, a security breach of a website with legitimate need for RWI could lead to greater consequences for identity theft or the trade of personal user information on illicit markets.
```

Comment (RByers) [reply risks.md]: Thanks Ori!

**Review by OR13 — APPROVED** [review]

**Review by kdenhartog — APPROVED** [review]
Good first start and makes sense to consolidate everything here

**Comment by RByers** [comment]
Thanks! Someone with write access want to merge it (or grant me write access)? It was @npdoty's suggestion to me on W3C #wicg-identity-cred slack to copy this over here.

**Comment by npdoty** [comment]
Thanks Rick and reviewers!

**Pull request merged**